validator/client/validator.go

```diff
@@ -564,7 +555,11 @@ func (v *validator) UpdateDuties(ctx context.Context) error {
 
 	v.dutiesLock.Lock()
 	v.duties = resp
-	v.logDuties(ss-params.BeaconConfig().SlotsPerEpoch, v.duties.CurrentEpochDuties, v.duties.NextEpochDuties)
+	ss, err := slots.EpochStart(epoch)
+	if err != nil {
+		return err
```

Comment:
Wouldn't it need to unlock here or something I don't think we should do this check here

Reply:
what check? for an error? that should never error